content/blog/2024-05-01-introducing-almalinux-hpc-ai-sig.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Introducing the AlmaLinux HPC and AI SIG"
+type: blog
+author: 
+ name: "Hayden Barnes"
+ bio: ""
+ image: /users/
```

Comment:
@sirredbeard we're missing a bio and user image for you here. Do you mind adding those? I can pull from public sources if not.

Reply:
Updated, looked at a couple other posts and didn't see any, but see some now